packages/pie-toolbox/src/code/plot/root.jsx

```diff
@@ -29,6 +29,8 @@ export class Root extends React.Component {
     rootRef: PropTypes.func,
     onChangeLabels: PropTypes.func,
     titlePlaceholder: PropTypes.string,
+    mathMlOptions: PropTypes.object,
+    charactersLimit: PropTypes.number,
```

Comment:
I suggest to rename this prop to `labelsCharactersLimit` in this file too

Reply:
Done